game.js

```diff
@@ -0,0 +1,48 @@
+const COMP_WINS = 0;
+const USER_WINS = 1;
+
+const computerSelection = computerPlay();
+console.log(computerSelection);
+const buttons = document.querySelector(".js-buttons-container"); //selected parent container
+buttons.addEventListener("click", choosePlayer); // adds listener on each clicked button
+
+function choosePlayer(e) {
+  const playerSelection = e.target.value;
+  console.log(playerSelection);
+  e.stopPropagation(); //Stop the event propagation from running up and down the DOM
```

Comment:
Stopping event propagation is a bad practice. There could be other event listeners defined in the DOM you don't know about. When you stop the event in your handler (listener), other listeners won't receive it.